ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/TestBlock.hs

```diff
@@ -550,7 +550,7 @@ data TestBlockLedgerConfig = TestBlockLedgerConfig {
   tblcHardForkParams :: HardFork.EraParams,
```

Comment:
Maybe also make this field strict for consistency? (Surprising that it doesn't seem to cause `NoThunks` failures...)